internal/usage/usage.go

```diff
@@ -219,6 +219,7 @@ const (
 	TargetProjectID                 = "Unique identifier of the project that contains the destination cluster for the restore job."
 	AccessListIPEntry               = "IP address to be allowed for a given API key."
 	AccessListIPEntries             = "List of IP addresses to be allowed to access the deployment."
+	AccessListIPEntryQuickstart     = "IP address to be allowed to access the deployment."
```

Comment:
[q] why not use `AccessListIPEntry`? they seem quite the same, also I would maybe rename `AccessListIPEntry` to `APIAccessListIPEntry` and `AccessListIPEntries` to `NetworkAccessListIPEntries` to avoid the same kind of mistake

Reply:
Because it says `List` where we are actually asking for a single entry

Reply:
Fair enought then let's follow my recomendation and make this clearer for all of us
`AccessListIPEntries` -> `NetworkAccessListIPEntries`
`AccessListIPEntry ` -> Api AccessListIPEntry`
`AccessListIPEntryQuickstart` -> `NetworkAccessListIPEntry`